demo/admin/src/common/EditPageNode.tsx

```diff
@@ -38,8 +38,16 @@ export const EditPageNode = createEditPageNode({
     },
     nodeFragment: additionalPageTreeNodeFieldsFragment,
     additionalFormFields: (
-        <div>
-            <Field label={<FormattedMessage id="pageTreeNode.fields.userGroup" defaultMessage="User-Group" />} name="userGroup" fullWidth>
+        <>
+            <Box marginY={6}>
+                <Divider />
+            </Box>
```

Comment:
You could also do `<Divider sx={{ marginY: 6 }}/>`. But this is fine.

Reply:
Ah okay. What does 'sx' stand for?

Reply:
I don't know 😅 It's a MUI feature: https://mui.com/system/getting-started/the-sx-prop/

Reply:
Ah okay interesting^^ Thanks 🙏🏻